### PR TITLE
Adapt to Lisk environment

### PR DESCRIPTION
**Pull request by clemente-xyz**
## Description

We want to update fork this repo, and update to the lisk branded with default network with lisk for the dev-rel effort.

## How was it solved?

- [x] Update the branding
- [x] Update the README
- [x] Update the default network

### How was it tested?

Manually.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-# 🏗🔴 Scaffold-OP
+# Scaffold-Lisk
 
 <h4 align="center">
   <a href="https://docs.scaffoldeth.io">Documentation</a> |
   <a href="https://scaffoldeth.io">Website</a>
 </h4>
 
-Scaffold-OP is a fork of Scaffold-ETH2 with minimal differences, providing additional dApp examples, native support for Superchain testnets, and more low-level instructions. We highly recommend the Scaffold-ETH2 docs as the primary guideline.
+Scaffold-Lisk is a fork of Scaffold-ETH2 with minimal differences, providing additional dApp examples, native support for Superchain testnets, and more low-level instructions. We highly recommend the Scaffold-ETH2 docs as the primary guideline.
```
Comment (shuse2) [inline README.md]: we are forking `Scaffold-OP`

**Review by shuse2 — CHANGES_REQUESTED** [review]
Screenshot 2024-08-12 at 11 27 11

* Links are not working
* Still Optimism logo on the README

File: README.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
-<div align="center">
-  <a href="https://optimism.io"><img alt="Optimism" src="https://raw.githubusercontent.com/ethereum-optimism/brand-kit/main/assets/svg/OPTIMISM-R.svg" width=320></a>
-  <br />
-  <br />
+<div align="left">
+  <a href="https://lisk.com"><img alt="Lisk" src="https://github.com/LiskHQ/scaffold-lisk/blob/main/packages/nextjs/public/readme-banner.png" width="100%"></a>
```
Comment (shuse2) [inline README.md]:
```suggestion
  <a href="https://lisk.com"><img alt="Lisk" src="./packages/nextjs/public/readme-banner.png" width="100%"></a>
```

File: README.md [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
 </div>
 
-# 🏗🔴 Scaffold-OP
+# Scaffold-Lisk
```
Comment (shuse2) [inline README.md]: with above image, it looks redundant

File: README.md [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
 
 <h4 align="center">
   <a href="https://docs.scaffoldeth.io">Documentation</a> |
   <a href="https://scaffoldeth.io">Website</a>
 </h4>
```
Comment (shuse2) [inline README.md]: these are link to the original, since it is mentioned below, we could remove it?

**Review by shuse2 — CHANGES_REQUESTED** [review]

**Review by shuse2 — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**